### ISSUE TITLE
Discussion Topic: Multiple Terminal (Frontend and backend

### ISSUE THREAD

**Issue by imabp** [alexa-dev-hub/alexa-community-jaipur#63]
@tarunnsingh I was working for a solution for it.

We dont need to do any stashing anymore.

Here is the solution

Clone the two branches in separate folder
using command

## For backend 

```shell

$git clone -b v1-deployment --single-branch https://github.com/<username>/alexa-community-jaipur

```

## For Frontend

```shell

$git clone -b master --single-branch https://github.com/<username>/alexa-community-jaipur

```

I think we should add this Branch Cloning to Docs

**Comment by imabp** [comment]
What you say @tarunnsingh . Is this good idea for now ?

**Comment by tarunnsingh** [comment]
> @tarunnsingh I was working for a solution for it.
> 
> We dont need to do any stashing anymore.
> 
> Here is the solution
> 
> Clone the two branches in separate folder
> using command
> 
> ## For backend
> ```shell
> $git clone -b v1-deployment --single-branch https://github.com/<username>/alexa-community-jaipur
> ```
> 
> ## For Frontend
> ```shell
> $git clone -b master --single-branch https://github.com/<username>/alexa-community-jaipur
> ```
> 
> I think we should add this Branch Cloning to Docs

Yes @imabp doing it this way is the only way which I told in #40 comments.
Nevertheless let's go with this arrangement, till we find a better fix.

Do you want to add this to Docs or shall I do it?

**Comment by imabp** [comment]
@tarunnsingh better if you put this to docs thing. I am checking the client side axios now

**Comment by tarunnsingh** [comment]
Okay then

**Comment by tarunnsingh** [comment]
Added #64

**Comment by imabp** [comment]
@tarunnsingh I have implemented the full Axios setup and had to put some modularity too. bit changes in project structure plus. 
Its working fine. 

But there is an issue, the image is still not sent by server. There is something in the backend part only which is obstructing it. Because front end part everything is getting fine.

**Comment by imabp** [comment]
Will send the PR regarding Axios setup soon
After giving final touches.

**Comment by imabp** [comment]
@tarunnsingh dont merge any pr until i send this  pr .. because i have done some major changes.

**Comment by tarunnsingh** [comment]
> @tarunnsingh I have implemented the full Axios setup and had to put some modularity too. bit changes in project structure plus.
> Its working fine.
> 
> But there is an issue, the image is still not sent by server. There is something in the backend part only which is obstructing it. Because front end part everything is getting fine.

Let me check.

**Comment by tarunnsingh** [comment]
> @tarunnsingh dont merge any pr until i send this pr .. because i have done some major changes.

okay sure

**Comment by imabp** [comment]
@tarunnsingh  This is making things not work. 

We need to send get request to  `api/posts/images/:imageName` which image we want to fetch. One more hook, we need then I think so.. 😥

![image](https://user-images.githubusercontent.com/53480076/95517870-804fc300-09df-11eb-94e4-c72581e45e10.png)

**Comment by imabp** [comment]
@tarunnsingh pls can you make all my PRs which are merged labelled as `hacktoberfest-accepted`.

Because they are not counted until and unless you put that label

![image](https://user-images.githubusercontent.com/53480076/95518113-ff44fb80-09df-11eb-824d-61ad696a7169.png)

**Comment by tarunnsingh** [comment]
> @tarunnsingh This is making things not work.
> 
> We need to send get request to `api/posts/images/:imageName` which image we want to fetch. One more hook, we need then I think so.. 😥
> 
> ![image](https://user-images.githubusercontent.com/53480076/95517870-804fc300-09df-11eb-94e4-c72581e45e10.png)

Yes this needs to be setup the same way as you made setup to fetch the blogposts (markdowns).
I was working on it locally too.

**Comment by tarunnsingh** [comment]
But I think if they are merged then they should be counted.
It has to be ONE of the 3 options.
Still Ill put up the labels if you say so.

**Comment by imabp** [comment]
@tarunnsingh Yes it showed a question mark on my PR on Hacktoberfest portal, but after you labelled, the question mark was gone.

Thank you @tarunnsingh  for such an overwhelming gratitude you said in that PR. I am obliged.

Here are final PRs which fixes image fetching completely  #70  #71 and also  pls label this PR too #69

**Comment by imabp** [comment]
@tarunnsingh I dont know what is going on here in this code. I am fetching the image correctly. The markdown images are being displayed properly no issues with that. But on those featured posts, it is not showing.

> But on a note, it is working on local implementation 

I am fetching from this link:
http://alexadevhub.herokuapp.com/api/posts/images/events/upcomingevent.png



But it is not displaying on the site
![image](https://user-images.githubusercontent.com/53480076/95597635-8344c500-0a6c-11eb-9de2-6b17c49db461.png)

**Comment by tarunnsingh** [comment]
Ya I saw that, let it be as such for now, I'll be making some more updates in coming days, including this one too.

**Comment by imabp** [comment]
Fine only those two images are having the issue, rest everything is fine.
